app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
         //noinspection OldTargetApi
         targetSdkVersion 26
         versionCode 1
```

Comment:
plz update versionCode as well! this is what the system uses to decide whether we need an upgrade.

I am seeing that this does not accept values of the form `1.0.1` however; maybe worth revisiting our numbering scheme?? Or we can use increasing values not related to the `versionName` for this value.

Reply:
>I am seeing that this does not accept values of the form 1.0.1 however

Right, because this should be an integer as I understand it. Like 3171 was the versionCode for versionName 1.0.0 (but versionCode is set by bitrise).

Perhaps I take the expected bitrise build number that would run when the next commit happens?

Reply:
Yeah that sounds reasonable!